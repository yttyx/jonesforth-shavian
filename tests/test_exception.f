( -*- 𐑑𐑧𐑒𐑑𐑕 -*- )

: 𐑑𐑧𐑕𐑑4 𐑐𐑮𐑦𐑯𐑑-𐑕𐑑𐑨𐑒-𐑑𐑮𐑱𐑕 𐑔𐑮𐑴 ;

: 𐑑𐑧𐑕𐑑3 0 𐑑𐑧𐑕𐑑4 26 𐑑𐑧𐑕𐑑4 ;

: 𐑑𐑧𐑕𐑑2
    ['] 𐑑𐑧𐑕𐑑3 𐑒𐑨𐑗
    ?𐑛𐑿𐑐 𐑦𐑓 ." 𐑑𐑧𐑕𐑑3 𐑔𐑮𐑿 𐑧𐑒𐑕𐑧𐑐𐑖𐑩𐑯 " . 𐑒𐑮 𐑞𐑧𐑯
    𐑑𐑧𐑕𐑑3
;

: 𐑑𐑧𐑕𐑑 𐑑𐑧𐑕𐑑2 ;
