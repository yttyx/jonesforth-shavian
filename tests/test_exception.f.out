𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+8 𐑒𐑨𐑗+28 𐑒𐑨𐑗 (  ) 𐑑𐑧𐑕𐑑2+8 𐑑𐑧𐑕𐑑+0 
𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+20 𐑒𐑨𐑗+28 𐑒𐑨𐑗 (  ) 𐑑𐑧𐑕𐑑2+8 𐑑𐑧𐑕𐑑+0 
𐑑𐑧𐑕𐑑3 𐑔𐑮𐑿 𐑧𐑒𐑕𐑧𐑐𐑖𐑩𐑯 26 
𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+8 𐑑𐑧𐑕𐑑2+108 𐑑𐑧𐑕𐑑+0 
𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+20 𐑑𐑧𐑕𐑑2+108 𐑑𐑧𐑕𐑑+0 
𐑳𐑯𐑒𐑷𐑑 𐑔𐑮𐑴 26 
